pkg/services/sqlstore/migrations/dashboard_public_config_mig.go

```diff
@@ -8,11 +8,9 @@ func addPublicDashboardMigration(mg *Migrator) {
 	var dashboardPublicCfgV1 = Table{
 		Name: "dashboard_public_config",
 		Columns: []*Column{
-			{Name: "uid", Type: DB_BigInt, IsPrimaryKey: true},
+			{Name: "uid", Type: DB_NVarchar, IsPrimaryKey: true},
 			{Name: "dashboard_uid", Type: DB_NVarchar, Length: 40, Nullable: false},
 			{Name: "org_id", Type: DB_BigInt, Nullable: false},
-			{Name: "refresh_rate", Type: DB_Int, Nullable: false, Default: "30"},
-			{Name: "template_variables", Type: DB_MediumText, Nullable: true},
```

Comment:
swooping in as a codeowner on DB migrations: are these columns used anywhere in the application? Just double checking as if so, they would not be safe to remove.

Reply:
@jalevin i'm not sure we should delete these options just yet as we do intend to use them eventually.

Reply:
We're going to drop and recreate the table, so we'll remove these and add them back once we have a use for them.